distuv/norm_example_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv_test
+
+import (
+	"fmt"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/stat"
+	"github.com/gonum/stat/distuv"
+)
+
+func ExampleNormal() {
+	const nmax = 1000
+
+	// Create a normal distribution
+	dist := distuv.Normal{
+		Mu:    2,
+		Sigma: 5,
+	}
+
+	data := make([]float64, nmax)
+
+	// Draw some random values from the standard normal distribution
+	for i := 0; i < nmax; i++ {
+		data[i] = dist.Rand()
+	}
+
+	mean, std := stat.MeanStdDev(data, nil)
+
+	const tol = 0.1
```

Comment:
I think more educational is to:
- Increase samples to 100,000
- Actually compute the error in mean and std and print them

````
mean, std := stat.MeanStdDev(data, nil)
meanErr := stat.StdErr(std, float64(nmax))
stdErr := std * math.Sqrt(2/(float64(nmax)-1))

fmt.Printf("mean = %1.1f +/- %0.1v\n", mean, meanErr)
fmt.Printf("std-dev = %1.1f +/- %0.1v\n", std, stdErr)

// Output:
// mean=    2.0 +/- 0.02
// std-dev= 5.0 +/- 0.02
````